cluster-chronological-split/readme.md

```diff
@@ -0,0 +1,16 @@
+# Cluster 10 split
+###Split dataset chronologically into a Training and Test dataset based on a Cluster
+
+* **Inputs:** Given a dataset chronologically ordered, a set of input fields, a K number and a split proportion
+* **Outputs:** Returns a dictionary including a test and training dataset resulting from the implemented split
+
+1. From the input dataset, trains a K-means Cluster (cluster fields and K as an input)
+2. Batch centroid over the original dataset to add K different cluster labels. This way data is separated in K groups of similarity
+3. For each cluster create a new dataset
+4. Split datasets linearly (using a proportion from the input parameter) keeping the latest for test
```

Comment:
Yes, I understand and am aware about this scenario. It's part of the solution and the user would need to be aware. The idea of implementing the script came from chemical plants where they have constant reactions 24/7/365 and they change the execution style now and then by modifying any of the parameters (many). It would be possible to have the scenario you mention about a cluster having only early and late data. That would not be a problem to split chronologically, it is the expected behavior from such solution. Even if training examples would be much older than test examples, by using this method we could ensure having examples of that cluster specific plant condition in both training and test datasets